tsconfig.build.json

```diff
@@ -11,6 +11,7 @@
     "rootDir": "src",
     "target": "ES2020",
     "esModuleInterop": true,
-    "ignoreDeprecations": "5.0"
+    "ignoreDeprecations": "5.0",
+    "moduleResolution": "node16"
```

Comment:
would this be considered a breaking change?

Reply:
I don't think this impacts the output code at all so don't think it's a breaking change.